README.md

```diff
@@ -13,22 +13,49 @@
+
+Jaseci presents a groundbreaking computational model that aims to simplify the process of developing AI applications. Through its exclusive data-spacial programming approach and diffuse execution environment, Jaseci has accomplished remarkable feats, such as decreasing development time by tenfold and almost completely removing standard backend code. Jaseci became open-source in 2021, and it is currently operational in four commercial products, including [Myca](https://www.myca.ai/), [HomeLendingPal](https://www.homelendingpal.com/), [ZeroShotBot](https://www.zeroshotbot.com/), and [TrueSelph](https://www.trueselph.com/), establishing itself as a pioneer in AI advancement.
+
+
+# Getting Started Guide Lines
+
+To install Jaseci and all its packages, refer to the comprehensive Jaseci [Installation Guide](https://docs.jaseci.org/docs/docs/getting_started/installation).
+
+To install Jaseci latest version in any linux based environment with `pip` use following command;
+
+```bash
+pip install jaseci
+```
+
+To set up a Jaseci server, please follow the steps outlined in the [Jaseci Server Setup guide](https://docs.jaseci.org/docs/docs/getting_started/setup_jaseci_serv).
+To set up Jaseci Studio, please follow the instructions outlined in the [following guide](https://docs.jaseci.org/docs/docs/getting_started/setting_up_jaseci_studio).
 
 # Write Your First Jaseci Application
-  - [(Optional) VSCode Extension](docs/docs/docs/getting_started/writing_first_app/setting_up_code_editor.md)
-  - [Building a Graph](docs/docs/docs/getting_started/writing_first_app/building_graph.md)
-  - [Develop Walkers and Abilities](docs/docs/docs/getting_started/writing_first_app/playing_with_walkers_and_abilities.md)
+
+```jac
+walker init{
+  std.out("Hello World");
+}
+```
+
+To run the hello world program, save the code in a `jac` file and run with `jac run` command in `jsctl` shell;
+
+```bash
+jaseci> jac run hello.jac
+```
+Follow these links to access more tutorials:
+
+- [(Optional) VSCode Extension](https://docs.jaseci.org/docs/docs/getting_started/writing_first_app/setting_up_code_editor)
+- [Building a Graph](https://docs.jaseci.org/docs/docs/getting_started/writing_first_app/buidling_graph)
+- [Develop Walkers and Abilities](https://docs.jaseci.org/docs/docs/getting_started/writing_first_app/playing_with_walkers_and_abilities)
 
 # Advanced Tutorials
-  - [Building a Conversational AI](docs/docs/tutorials/CanoniCAI/1_preparation.md)
-  - [Building a Text Analytics System](docs/docs/tutorials/CanoniAnalytics/1_preparation.md)
+  - [Building a Conversational AI](https://docs.jaseci.org/docs/category/build-a-conversational-ai-system-with-jaseci)
```

Comment:
Is there a reason that you want use URL? instead of relative link to the markdown files.